osu.Game/Screens/Edit/Compose/Components/ComposeBlueprintContainer.cs

```diff
@@ -54,8 +56,38 @@ protected override void LoadComplete()
             base.LoadComplete();
 
             inputManager = GetContainingInputManager();
+
+            Beatmap.SelectedHitObjects.CollectionChanged += (_, __) => updateTogglesFromSelection();
```

Comment:
This will trigger the hitobjects to update too:
CollectionChanged --> NewCombo.Value = X --> NewCombo.ValueChanged --> c.NewCombo = X

https://drive.google.com/file/d/102MVgd3qmAV9X-qwIvCrIWPifOwYJdeO/view?usp=sharing

Reply:
Hmm, able to overlook this as it's fixed in the next PR (#10243)?